python/cudf/cudf/io/parquet.py

```diff
@@ -824,6 +966,38 @@ def read_parquet(
     # Normalize and validate filters
     filters = _normalize_filters(filters)
 
+    if (
+        engine == "cudf"
+        and filters is not None
+        and categorical_partitions is None
+        and dataset_kwargs is None
+        and len(filepath_or_buffer) == 1
```

Comment:
Do libcudf parquet filters not work with multiple files?

Reply:
They do, removed the guard